includes/class-wc-payments-account.php

```diff
@@ -2161,6 +2174,12 @@ private function finalize_connection( string $state, string $mode, array $additi
 		$gateway->update_option( 'enabled', 'yes' );
 		$gateway->update_option( 'test_mode', 'live' !== $mode ? 'yes' : 'no' );
 
+		$capabilities = $this->onboarding_service->get_capabilities_from_request();
```

Comment:
Please add more inline docs here explaining where these capabilities come from and why we enforce them.

Reply:
Included an explanation with inline documentation.